content/docs/alerting/rules.md

```diff
@@ -15,53 +15,63 @@ Alerting rules are configured in Prometheus in the same way as [recording
 rules](../../querying/rules).
 
 ### Defining alerting rules
+
 Alerting rules are defined in the following syntax:
 
     ALERT <alert name>
       IF <expression>
       [FOR <duration>]
-      [WITH <label set>]
-      SUMMARY "<summary template>"
-      DESCRIPTION "<description template>"
+      [LABELS <label set>]
+      [ANNOTATIONS <label set>]
 
 The optional `FOR` clause causes Prometheus to wait for a certain duration
 between first encountering a new expression output vector element (like an
 instance with a high HTTP error rate) and counting an alert as firing for this
 element. Elements that are active, but not firing yet, are in pending state.
 
-The `WITH` clause allows specifying a set of additional labels to be attached
-to the alert. Any existing conflicting labels will be overwritten.
+The `LABELS` clause allows specifying a set of additional labels to be attached
+to the alert. Any existing conflicting labels will be overwritten. The label
+values can be templated.
+
+The `ANNOTATIONS` clause specifies another set of labels that are not
+identifying for an alert instance. They are used to store longer additional
+information such as alert descriptions or runbook links. The annotation values
+can be templated.
+
+#### Templating
 
-The `SUMMARY` should be a short, human-readable summary of the alert (suitable
-for e.g. an email subject line), while the `DESCRIPTION` clause should provide
-a longer description. Both string fields allow the inclusion of template
-variables derived from the firing vector elements of the alert:
+Label and annotation values can be templated using Go's template language.
```

Comment:
This should have a pointer over to the template reference, as it's not purely Go's template language.